webpack.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+    entry: [ "./public/scripts/main.js","./public/scripts/keymaster.js" ],
```

Comment:
Not sure if you missed in my original comments, but I confirmed this only needs to be ` entry: "./public/scripts/main.js",` since we should require keymaster from main.